packager/datapackage_descriptor.json

```diff
@@ -86,243 +95,232 @@
                     }
                 },
                 {
-                    "name": "worklplace",
-                    "title": "Workplace",
+                    "name": "lotacao",
+                    "title": "Lotacao",
                     "type": "string",
                     "description": "Employee workplace",
                     "format": "default",
                     "description-ptbr": "O local onde o empregado está alocado"
                 },
                 {
-                    "name": "active",
-                    "title": "Active",
+                    "name": "em_atividade",
+                    "title": "Em atividade",
                     "type": "boolean",
                     "description": "Active or inactive employee",
                     "description-ptbr": "Descreve se o funcionário está ativo ou inativo"
```

Comment:
Adicionar constraint de required?

Reply:
Nao adicionei porque o [TJPB](https://www.tjpb.jus.br/sites/default/files/anexos/2020/04/transparencia_202003_magistrados_0.pdf) nao especifica por exemplo se está em atividade ou nao. Lembro que dissestes que outra entidade era responsável por tratar dos pensionistas. Nesse caso eu nao sei bem o que fazer também, o que me diz?

Reply:
Acho que temos que entender melhor os dados disponibilizados pelo TJPB (quer seja via site ou pedido de informação). 

Entender se  realmente todos que estão no site são ativos. Se for, temos que ter um comentário gigante no coletor explicando que todos são ativos e que ainda não coletamos os inativos. Somado a isso, temos que abrir uma isso para procurar os inativos do TJPB.

cc/ @nazareno